storage/plugins/archives/ArchivesPlugin.php

```
@@ -3983,6 +4035,858 @@ public function plannedHooks(): array
+        $xw->writeElement('adminEmail', $adminMail);
+        $xw->writeElement('earliestDatestamp', $earliest);
+        $xw->startElement('deletedRecord');
+        $xw->text('no');
+        $xw->endElement();
+        $xw->startElement('granularity');
+        $xw->text('YYYY-MM-DDThh:mm:ssZ');
+        $xw->endElement();
+        $xw->endElement(); // Identify
+    }
+
+    private function oaiListMetadataFormats(\XMLWriter $xw): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $xw. Configured minimum length is 3. (undefined)

(ShortVariable)